package.json

```diff
@@ -38,13 +38,11 @@
   "lint-staged": {
     "*.{js,json}": [
       "eslint --fix",
-      "prettier --write",
-      "git add"
+      "prettier --write"
     ],
     "*.vue": [
       "prettier --write",
-      "eslint --fix",
-      "git add"
```

Comment:
Why you have removed that? Without it, formatted files would not be commited

Reply:
after update husky to 4.x.x version, no need to call `git add` cmd.
i have warning message about it when start committing

Reply:
👌